doc/nrf/releases/release-notes-latest.rst

```diff
@@ -187,6 +294,23 @@ The following list summarizes the most important changes inherited from upstream
     * Unified endpoint helper macros across all USB device drivers.
     * Fixed handling of fragmented transfers on the control OUT endpoint in the Nordic Semiconductor USB Device Controller driver (usb_dc_nrfx).
     * Introduced names for threads used in USB classes, to aid debugging.
+    * Updated the way the :cpp:func:`usb_enable` function should be used.
+      For some samples, this function was invoked automatically on system boot-up to enable the USB subsystem, but now it must be called explicitly by the application.
+      If your application relies on any of the following Kconfig options, it must also enable the USB subsystem:
+
+      * :option:`CONFIG_OPENTHREAD_NCP_SPINEL_ON_UART_ACM`
+      * :option:`CONFIG_USB_DEVICE_NETWORK_ECM`
+      * :option:`CONFIG_USB_DEVICE_NETWORK_EEM`
+      * :option:`CONFIG_USB_DEVICE_NETWORK_RNDIS`
+      * :option:`CONFIG_TRACING_BACKEND_USB`
+      * :option:`CONFIG_USB_UART_CONSOLE`
+
+    * Updated the implementation of the CDC class to not duplicate USB transfer slots for OUT transfers.
+    * Updated the implementation of CDC ACM to re-trigger transfer for the OUT endpoint on resume.
```

Comment:
The first one is fixing an issue introduced by the second one. If we need to say about all bug fixes then please concatenate those two into:
`Fix an issue that CDC ACM was not accepting OUT transfers after Resume from Suspend.`